dart/lib/src/enricher/web_enricher_event_processor.dart

```diff
@@ -84,4 +90,16 @@ class WebEnricherEventProcessor extends EventProcessor {
     }
     return null;
   }
+
+  Map<String, dynamic> _getDartContext() {
+    final compile_mode = _options.platformChecker.isReleaseMode()
+        ? 'release'
+        : _options.platformChecker.isDebugMode()
+            ? 'debug'
+            : 'profile';
```

Comment:
We could extract that within the `platformChecker` then so we don't duplicate this part of the code

Reply:
Done